x-pack/plugins/apm/public/components/app/tutorials/commands/node.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import {
+  serviceNameHint,
+  secretTokenHint,
+  serverUrlHint,
+  serviceEnvironmentHint,
+  apiKeyHint,
+} from './shared_hints';
+
+export const nodeVariables = {
+  apmServiceName: 'serviceName',
+  secretToken: 'secretToken',
+  apiKey: 'apiKey',
+  apmServerUrl: 'serverUrl',
+  apmEnvironment: 'environment',
+};
+
+export const nodeHighlightLang = 'js';
+
+const nodeServiceNameHint = i18n.translate(
+  'xpack.apm.tutorial.nodeClient.createConfig.commands.serviceName',
+  {
+    defaultMessage: 'Overrides the service name in package.json.',
+  }
+);
+
+export const nodeLineNumbers = (apiKey?: string) => ({
+  start: 1,
+  highlight: '3, 5, 7, 9',
+  annotations: {
+    3: `${serviceNameHint} ${nodeServiceNameHint}`,
+    5: apiKey ? apiKeyHint : secretTokenHint,
+    7: serverUrlHint,
+    9: serviceEnvironmentHint,
+  },
+});
```

Comment:
This feels quite complex. Did the agent instructions require this before? It should be as easy as possible for agent developers (who are not familiar with javascript or Kibana) to contribute to this.

Reply:
Just for comparison, I think we should strive for this level of simplicity (just with dynamic values for api keys and such)

This asciidoc:
https://github.com/elastic/observability-docs/blob/main/docs/en/shared/install-apm-agents/content.asciidoc

Is rendered as this content:
https://www.elastic.co/guide/en/apm/guide/current/apm-quick-start.html#add-apm-integration-agents

Reply:
Well yes, this was present in the previous agent instructions. We need the Line Numbers and Annotations for all agents to render an informative code block.
![image](https://github.com/elastic/kibana/assets/7416358/6d15bb85-f7c4-42bb-8c5a-0c28a86a8be6)

But i like your proposal as well, instead of rendering annotations with Hints, we can use `code comments` as shown in the asciidocs which i feel is more easier to maintain and contribute to. I will change it now

Reply:
@sqren This is how it will look like if we simplify it to code comments


https://github.com/elastic/kibana/assets/7416358/bad4d88f-174c-45de-83a0-2bc58c64977c

Reply:
Thanks @achyutjhunjhunwala ! I also prefer code comments.
Please bring this up at next sync where the PMs are there (Monday or Wed) so we can get their take on it.